src/cpp/session/modules/SessionMarkers.hpp

```diff
@@ -32,6 +33,7 @@ namespace modules {
 namespace markers {
 
 core::json::Object markersStateAsJson();
+std::vector<module_context::SourceMarker> markersForFile(std::string path);
```

Comment:
nit: prefer using `const&` for non-primitive input parameters that are read-only

```suggestion
std::vector<module_context::SourceMarker> markersForFile(const std::string& path);
```